znai-docs/znai/flow/meta.json

```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Znai",
+  "type": "User Guide",
+  "category": "Documentation",
+  "description": "Transforms Markdown files into rich, well-structured documentation served in a modern web front-end. Znai provides many custom add-ons that supercharge basic generated docs with advanced layout features, dynamic references to code and test artifacts, and many visual and functional enhancements.",
+  "viewOn": {
+    "link": "https://github.com/testingisdocumenting/znai/blob/master/znai-docs/znai",
+    "title": "View On GitHub"
+  },
+  "support": {
+    "link": "https://testingisdocumenting.slack.com/app_redirect?channel=help-znai"
```

Comment:
is this a real link? I don't think it will work for open source hosted doc...

Reply:
The link is a fake one. But if a user puts a real link, it should work. 
The link will be converted
    
    <a href="https://testingisdocumenting.slack.com/app_redirect?channel=help-znai" target="_blank" />

Reply:
Ah I didn't realize it is NOT znai meta.json file is a separate file. 
Should we add a support link to the real one and give it a link to GitHub issues?

Reply:
added.

Reply:
thank you. shall we delete this file and use the znai meta.json itself?